sunbird-platform-core/common-util/src/main/java/org/sunbird/common/request/RequestValidator.java

```diff
@@ -40,62 +40,59 @@ private RequestValidator() {}
    */
   @SuppressWarnings("unchecked")
   public static void validateUpdateContent(Request contentRequestDto) {
-    if (((List<Map<String, Object>>) (contentRequestDto.getRequest().get(JsonKey.CONTENTS)))
-        .isEmpty()) {
-      throw new ProjectCommonException(
-          ResponseCode.contentIdRequired.getErrorCode(),
-          ResponseCode.contentIdRequiredError.getErrorMessage(),
-          ERROR_CODE);
-    } else {
+
+    List<Map<String, Object>> contents =  (List<Map<String, Object>>) contentRequestDto.getRequest().get(JsonKey.CONTENTS);
+    if (!CollectionUtils.isEmpty(contents)) {
       List<Map<String, Object>> list =
-          (List<Map<String, Object>>) (contentRequestDto.getRequest().get(JsonKey.CONTENTS));
+              (List<Map<String, Object>>) (contentRequestDto.getRequest().get(JsonKey.CONTENTS));
       for (Map<String, Object> map : list) {
         if (null != map.get(JsonKey.LAST_UPDATED_TIME)) {
           boolean bool =
-              ProjectUtil.isDateValidFormat(
-                  "yyyy-MM-dd HH:mm:ss:SSSZ", (String) map.get(JsonKey.LAST_UPDATED_TIME));
+                  ProjectUtil.isDateValidFormat(
+                          "yyyy-MM-dd HH:mm:ss:SSSZ", (String) map.get(JsonKey.LAST_UPDATED_TIME));
           if (!bool) {
             throw new ProjectCommonException(
-                ResponseCode.dateFormatError.getErrorCode(),
-                ResponseCode.dateFormatError.getErrorMessage(),
-                ERROR_CODE);
+                    ResponseCode.dateFormatError.getErrorCode(),
+                    ResponseCode.dateFormatError.getErrorMessage(),
+                    ERROR_CODE);
           }
         }
         if (null != map.get(JsonKey.LAST_COMPLETED_TIME)) {
           boolean bool =
-              ProjectUtil.isDateValidFormat(
-                  "yyyy-MM-dd HH:mm:ss:SSSZ", (String) map.get(JsonKey.LAST_COMPLETED_TIME));
+                  ProjectUtil.isDateValidFormat(
+                          "yyyy-MM-dd HH:mm:ss:SSSZ", (String) map.get(JsonKey.LAST_COMPLETED_TIME));
           if (!bool) {
             throw new ProjectCommonException(
-                ResponseCode.dateFormatError.getErrorCode(),
-                ResponseCode.dateFormatError.getErrorMessage(),
-                ERROR_CODE);
+                    ResponseCode.dateFormatError.getErrorCode(),
```

Comment:
dateFormatError can be in one place , as of now same code is above as well.

Reply:
This was existing code. We didn't change anything.

Reply:
but it can be updated(: